torch/quantization/ns/graph_matcher.py

```diff
@@ -185,16 +246,18 @@ def _node_a_related_to_b(
         return key in type_a_related_to_b
     return False
 
-def _get_name_for_node_pair(
-    node_a: Node,
-    node_b: Node,
+def _get_name_for_subgraph_pair(
+    start_node_a: Node,
+    end_node_a: Node,
+    start_node_b: Node,
+    end_node_b: Node,
 ) -> str:
-    if node_b.op == 'call_module':
-        assert isinstance(node_b.target, str)
-        return node_b.target
+    if end_node_b.op == 'call_module':
+        assert isinstance(end_node_b.target, str)
+        return end_node_b.target
     # for now, use node name.
     # TODO(future PR): find a better solution
-    return node_b.name
+    return end_node_b.name
```

Comment:
Should this be start_node_b.name+end_node_b.name so that it is clear that it is a sub-graph and not a single node?

Reply:
Sounds good to me.  In general naming of nodes is something we should talk through in detail for all of these APIs, saving that for a future PR.  Thankfully changing that later should be low eng cost, so it doesn't have to be perfect now.